plasmapy/classes/sources/plasma3d.py

```diff
@@ -6,15 +6,9 @@
 import astropy.units as u
 import itertools
 
-from plasmapy.physics.magnetostatics import MagnetoStatics
-
-from astropy.constants import (m_p,
-                               m_e,
-                               c,
-                               mu0,
-                               k_B,
-                               e,
-                               eps0,
+from plasmapy.formulary.magnetostatics import MagnetoStatics
+
+from astropy.constants import (mu0,
                                )
```

Comment:
Auto-formatting shenanigans? Newline seems a bit excessive with a single import...

Reply:
Curse you, PyCharm! Thanks, Drew! :)